app/views/publish/courses/about_this_course/edit.html.erb

```diff
@@ -0,0 +1,71 @@
+<% content_for :page_title, title_with_error_prefix(
+  t("publish.providers.about_course.edit.page_title", course_name_and_code: @course.name_and_code),
+  @about_this_course_form.errors.any?
+) %>
+
+<% if params[:copy_from].present? && @copied_fields.any? %>
+  <%= render partial: "copy_about_course_content_warning" %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+          model: @about_this_course_form,
+          url: about_this_course_publish_provider_recruitment_cycle_course_path(
+            @provider.provider_code,
+            @course.recruitment_cycle_year,
+            @course.course_code
+          ),
+          data: { qa: "enrichment-form", module: "form-check-leave" },
+          method: :patch,
+          local: true
+        ) do |f| %>
+
+      <% content_for :before_content do %>
+        <%= govuk_back_link_to(
+              back_link_path(param_form_key: f.object_name.to_sym,
+                             params:,
+                             provider_code: @provider.provider_code,
+                             recruitment_cycle_year: @course.recruitment_cycle_year,
+                             course_code: @course.course_code)
+            ) %>
+      <% end %>
+
+      <%= f.govuk_error_summary %>
```

Comment:
This should be full width: 

<img width="895" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/a0de1dce-dec9-4c25-bb3d-1acd26a27630">


Similar to this: 

<img width="1006" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/cb5060ee-5234-465c-8890-530ffecbcbbd">

Reply:
hmmm.... good question. I'll make the change

Reply:
Actually, the error component needs to sit within the form I think.

Reply:
Also, it's not how it works on any of the existing pages. I think if we want to change it, it should be done in a separate ticket and fix throughout.